eva/executor/orderby_executor.py

```diff
@@ -65,14 +67,26 @@ def exec(self) -> Iterator[Batch]:
             aggregated_batch_list.append(batch)
         aggregated_batch = Batch.concat(aggregated_batch_list, copy=False)
 
+        # Column can be a functional expression, so if it
+        # is not in columns, it needs to be re-evaluated.
+        merge_batch_list = [aggregated_batch]
+        for col in self._columns:
+            if col.col_alias not in aggregated_batch.frames:
+                batch = col.evaluate(aggregated_batch)
+                merge_batch_list.append(batch)
+        if len(merge_batch_list) > 1:
```

Comment:
We have a merge function in batch. Can we reuse it here?

Reply:
Fix this.